docs/pipelines/trainable/ner.md

```diff
@@ -0,0 +1,117 @@
+# Nested Named Entity Recognition
+
+The default spacy Named Entity Recognizer (NER) pipeline only allows flat entity
```

Comment:
Can you search mentions of `spacy` and replace them with `spaCy` when needed? Sorry for the chore...

```suggestion
The default spaCy Named Entity Recognizer (NER) pipeline only allows flat entity
```